### ISSUE TITLE
produce preparatory communication to participants

### ISSUE THREAD

**Issue by tomkralidis** [geopython/geopython-workshop#21]
Participants should receive communications on workshop requirements, what to bring/how to setup so that when they arrive to the workshop they are not installing Docker, etc.

Suggest this is best done on https://geopython.github.io/geopython-workshop as the canonical location for communication, which can be referenced accordingly in email, etc.

- [x] update instructions on `gh-pages` (@tomkralidis / @justb4 / @ldesousa / @kalxas)
- [x] coordinate with FOSS4G 2019 organizers (@tomkralidis)

**Comment by justb4** [comment]
and https://gitter.im/geopython/geopython-workshop for comms, or would we like to keep that for dev?

**Comment by tomkralidis** [comment]
YI I put back a simpler mkdocs setup in https://github.com/geopython/geopython-workshop/tree/master/web with output in https://geopython.github.io/geopython-workshop/

tl; dr: edit content in `web/docs`, commit, then push to live site with `mkdocs gh-deploy -m 'some message'` (see `web/README.md`).  Note you still have to push the source changes to master before pushing to live site with `mkdocs gh-deploy`.

Note that content is first pass for review/update by others.
